config/locales/en.yml

```diff
@@ -486,3 +486,11 @@ en:
     transparency: See all our transparency data
     written_statement: See all our written statements to Parliament
     authored_article: See all our authored articles
+  broken_links:
+    broken: 
+      title: Broken links
+      subheading: There are broken links in this document
+    caution:
+      title: Links with warnings
```

Comment:
I don't think it's necessary to have a title on the warning, I think it would be better something like:

# Broken links
> We’ve found links in this document that may be broken:

- url
- ...

> We've found some issues with links that may indicate problems, you should apply caution in linking to them:

- url
- ...

Similar to the design in https://trello.com/c/qKlOIUfB/36-make-whitehall-reports-nice-for-users